docs/aws-athena.md

```diff
@@ -0,0 +1,67 @@
+# AWS Athena integration setup
+
+## Pre-requisites
+
+1. An AWS Athena S3 data-source
+2. An AWS Glue crawler and database
+3. The AWS Glue crawler has been run at least once to populate the tables in the database
+
+## Usage
+
+In the example below, it is assumed that the [NCBI SRA Metadata](https://www.ncbi.nlm.nih.gov/sra/docs/sra-athena/) has been used as the data source. You can refer the [tutorial from NCBI](https://www.youtube.com/watch?v=_F4FhcDWSJg&ab_channel=TheNationalLibraryofMedicine) for setting up the AWS resources correctly
+
```

Comment:
I thought about adding the AWS CLI commands to get this done, but then we end up owning the maintenance of the docs so I delegated to the NCBI folks a public database.